README.md

```diff
@@ -356,3 +357,49 @@ defaulted in the `entrypoint.sh` script.
 |K2HB_AWS_S3_PARALLEL_PUTS| If not batch putting allows all messages to be written in parallel - n.b. may cause rate limiting issues |
 |K2HB_AWS_S3_REGION| Set this if eu-west-2 is not where you want to point |
 |K2HB_AWS_S3_USE_LOCALSTACK| Set to true for local running - uses containerized aws|
+
+# Sample 101 Kafka command lines for practice
+
+Bring up all the service containers and get a shell in the kafka box
+   ```shell script
+   make services
+   make kafka-shell
+   ```
+
+Inside the shell, find all the utility scripts
+   ```shell script
+   cd /opt/kafka/bin
+   ls
+   ```
+
+Check the current list of topics
+   ```shell script
+   ./kafka-topics.sh --zookeeper zookeeper:2181 --list
+   ```
+
+Make a new topic
+...note that doing it this way we must specify the partitions, while through code it is defaulted at the server level.
```

Comment:
this renders a bit funny, did you want multiple lines?

Reply:
Ahh yes, thx